lib/newsUtils.js

```diff
@@ -0,0 +1,27 @@
+import BASE from './common';
+
+function createNewsItemData(newsItem) {
+  return {
+    date: new Date(newsItem.get('Created')),
+    description: newsItem.get('Description'),
+    title: newsItem.get('Title'),
+    id: newsItem.get('ID')
+  };
+}
+
+const getNewsItems = function async() {
+  return BASE('News')
+    .select({
+      view: 'Grid view',
+      sort: [{ field: 'Created', direction: 'desc' }]
```

Comment:
Also, realizing this now...you could just use the `getAllNews()` function and sort it locally?

Reply:
This is a great point (-: I still think it'd be nice to have optional sort functions but it doesn't matter in the grand scheme of things 😂